packages/webpack-loader/test/typescript-loader.unit.ts

```diff
@@ -35,6 +36,32 @@ describe('webpack loader', () => {
             expect(stats.hasErrors(), statsText).to.equal(false);
             expect(stats.hasWarnings(), statsText).to.equal(false);
         });
+
+        it(`allows ts.transformers to be used`, async () => {
+            let transpileCtx!: ts.TransformationContext;
+            const entry = join(fixturesRoot, 'transformed.ts');
+            const { stats, statsText } = await bundleWithLoader({
+                entry,
+                options: {
+                    compilerOptions: {
+                        jsx: 'preserve'
+                    },
+                    cache: false,
+                    transformers: {
+                        before: [
+                            ctx => node => {
+                                transpileCtx = ctx;
+                                return node;
+                            }
+                        ]
+                    }
+                }
+            });
+
+            expect(stats.hasErrors(), statsText).to.equal(false);
+            expect(stats.hasWarnings(), statsText).to.equal(false);
+            expect(transpileCtx?.getCompilerOptions()).not.to.be.undefined;
```

Comment:
`to.not.equal(undefined)`.
I'm trying to avoid getter-based matchers, as you often get no indication when you think something is a matcher, but its not.